data/beaconrestapi/src/test/java/tech/pegasys/artemis/beaconrestapi/BeaconRestApiTest.java

```diff
@@ -31,57 +33,51 @@
 import tech.pegasys.artemis.storage.ChainStorageClient;
 
 class BeaconRestApiTest {
-  private final ChainStorageClient storageClient =
+  private static final ChainStorageClient storageClient =
       ChainStorageClient.memoryOnlyClient(new EventBus());
```

Comment:
This will share the client and event bus between tests which seems like a bad idea.

Reply:
fixed